tests/integration/targets/k8s_hide_fields/tasks/main.yml

```diff
@@ -86,6 +87,22 @@
         that:
           - hf6.changed
 
+    - name: Ensure hidden fields are not present
+      assert:
+        that:
+          - >-
+            'annotations' not in hf6.result.metadata or
```

Comment:
I'm curious why the `or` is needed in this set of test assertions. Are there cases where this same test would sometimes return a `metadata.annotations` field in the result and sometimes not?

Reply:
I think the intent here is to avoid the test failing because the `annotations` field has been hidden when it contains only `kubectl.kubernetes.io/last-applied-configuration` key (the list of config map annotations may depend on the cluster/resource version)